data-migration/migrate-sharding-mysql-tidb-less-tb.md

```diff
@@ -0,0 +1,239 @@
+---
+title: Migrate and Merge MySQL Shards of Less Than 1 TiB to TiDB
+summary: Learn how to consolidate MySQL shards of less than 1 TiB to TiDB.
+---
+
+# Migrate and Merge MySQL Shards of Less Than 1 TiB to TiDB
+
+If you want to migrate and merge multiple MySQL database instances upstream to one TiDB database downstream, and the amount of data is not too large (for example, the sum of all MySQL shards is less than 1 TiB), you can use DM to migrate MySQL shards. Through examples in this document, you can learn the operation steps, precautions, and troubleshooting of the migration.
+
+This document applies to migrating MySQL shards less than 1 TiB in total. If you want to migrate MySQL shards with a total of more than 1 TiB of data, it will take a long time by using DM. In this case, it is recommended that you follow the operation introduced in [Migrate and Merge MySQL Shards of More Than 1 TiB to TiDB](/migrate-sharding-mysql-tidb-above-tb.md) to perform migration.
```

Comment:
```suggestion
This document applies to migrating MySQL shards less than 1 TiB in total. If you want to migrate MySQL shards with a total of more than 1 TiB of data, it will take a long time by using DM. In this case, it is recommended that you follow the operation introduced in [Migrate and Merge MySQL Shards of More Than 1 TB to TiDB](/migrate-sharding-mysql-tidb-above-tb.md) to perform migration.
```

Reply:
避免用户当成 Typo 错误，我们还是打算吧标题中的 TiB 改为 TB.......